ui/component/joinMembershipCard/internal/previewPage/internal/membershipTier/index.jsx

```diff
@@ -0,0 +1,37 @@
+// @flow
+import React from 'react';
+import * as ICONS from 'constants/icons';
+import Button from 'component/button';
+import MembershipDetails from '../membershipDetails';
+
+type Props = {
+  key?: any,
+  membership: CreatorMembership,
+  disabled?: boolean,
+  handleSelect: () => void,
+};
+
+const MembershipTier = (props: Props) => {
+  const { key, membership, disabled, handleSelect } = props;
```

Comment:
:small_red_triangle: The key property is used by React under the hood, and is not exposed to you. 

<img width="522" alt="image" src="https://user-images.githubusercontent.com/64950861/190139312-0d9e15fb-bc8f-4739-9d77-1cff3a5d4a5c.png">